addons/account/models/chart_template.py

```diff
@@ -159,8 +159,8 @@ def _notify_accountant_managers(taxes_to_check):
             )
 
     env = api.Environment(cr, SUPERUSER_ID, {})
-    chart_template_id = env['ir.model.data']._xmlid_to_res_id(chart_template_xmlid)
-    companies = env['res.company'].search([('chart_template_id', '=', chart_template_id)])
+    chart_template_id = env.ref(chart_template_xmlid, raise_if_not_found=False)
+    companies = env['res.company'].search(['|', ('chart_template_id', '=', chart_template_id), ('chart_template_id', 'child_of', chart_template_id)])
```

Comment:
@dylankiss I am afraid we need .id here.  Would be good to check in 14 as well.  https://runbot.odoo.com/runbot/build/23000729

Reply:
The _xmlid_to_res_id probably really returned an id, while ref returns a browse record